services/movie.py

```diff
@@ -0,0 +1,35 @@
+from db.models import Movie
+from django.db.models import QuerySet
+from typing import List, Optional
```

Comment:
Please sort imports: built-in is first, third-party next, you own imports is the last. Blank line between each block